angular/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: "3.4"
```

Comment:
I assume this file was just created for testing the Dockerfile during development? If so it might not be necessary anymore